packages/transports/http-callback/src/index.ts

```diff
@@ -138,8 +138,10 @@ export default {
           stopSubscription(createTimeoutError());
         }, heartbeatIntervalMs),
       );
-      subscriptionLogger?.debug(
-        `Subscribing to ${transportEntry.location} with callbackUrl: ${callbackUrl}`,
+      log.debug(
+        'Subscribing to %s with callbackUrl: %s',
```

Comment:
What is the difference between this and template literals?

Reply:
I've [added benchmarks](https://github.com/graphql-hive/gateway/pull/1030/commits/2c512a9004f7e2ac471bde15cf9dd9b064d6f87e), the overhead is minimal (template literals being on average 1.10x faster).

However, using the interpolation approach has 2 benefits:
- You have useful placeholders like `%o` for formatting objects right there
- It's easier to read because the placeholders are shorter therefore having the whole log message right there

Nevertheless, it's just a convention you'll see out there. We can switch back to using the literals in the gateway if you want (even though we'll keep the interpolation feature).

Reply:
I am just afraid the overhead/latency might grow silently (exponentially also) with "safe" stringifying and formatting phases. On the other hand, we don't usually print obejcts that often especially by default